script/Ciel/Day 7/07_01_CIEL07_1B.txt

```diff
@@ -207,24 +208,26 @@ She even starts clapping her hands. She's totally just making fun of me.
 [sha:e086f0f496595a2437e62096f68415920b92f423]{
 -- Page 47, Offset 14921.
 -- そのあまりにもとぼけた反応で、俺の頭もようやく落ち着いてくれた。
-I finally start to calm down from seeing how flagrantly she's playing dumb.
+Faced with this flagrant display of feigned ignorance, I finally start to calm down.
 }
 [sha:b05a6e4b64139d03fca01465c4082b3778aec857]{
 -- Page 48, Offset 14922.
 -- 「……もういいです。どうせ俺なんて先輩の中じゃどうでもいい扱いなんだろうし。騒いですみませんでした」
-"Whatever... I guess Senpai just doesn't give a crap about me.
- Sorry for getting so worked up."
+"Fine, whatever...
+ Looks like you doesn't give a crap about me after all.
+ Sorry for bothering you."
 }
 [sha:157bb6b3d11473535b9384179e928cdc159051a9]{
 -- Page 49, Offset 14923.
 -- 「ぁ……いえ、そう言われると、その……
-"Oh, um. When you put it like that, I――
+"Oh, um... When you put it like that, well...
 }
 [sha:876672a3f688edf5d6de280ad27ba03ba06d4d9a]{
 -- Page 49, Offset 14924.
 -- ごめんなさい、悪ふざけが過ぎました。遠野くんが予想以上に驚いてくれるから、つい」
-I apologize. I took my teasing too far.
- It's just, seeing how surprised you got, I couldn't help it."
+// Raw 'surprised' doesn't really capture what I'd describe his behaviour as
```

Comment:
I don't quite follow; he's clearly surprised to see her, that's why he's acting so flustered.
This seems like a odd change to me.

Reply:
I think he's saying that "surprise" is too weak a word to describe his behavior, not that he's not surprised. New TL is definitely more smooth.

Reply:
Yeah, to me it was a question of what it's responding to - he's surprised to _see her_, sure, but it feels weirder to say he's surprised as a result of her teasing, which is the subject here.